### ISSUE TITLE
Stopped starting

### ISSUE THREAD

**Issue by nanobyte84** [lendy007/homeassistant-skodaconnect#16]
Hi, since a few days the integrations stopped working and on startup it takes ages and slows down the full Home Assistant:

2021-02-16 21:38:45 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: skodaconnect
2021-02-16 21:38:46 WARNING (MainThread) [homeassistant.setup] Setup of skodaconnect is taking over 10 seconds.
2021-02-16 21:39:45 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: skodaconnect

Logger: homeassistant.setup
Source: setup.py:215
First occurred: 21:43:36 (1 occurrences)
Last logged: 21:43:36

Setup of skodaconnect is taking longer than 300 seconds. Startup will proceed without waiting any longer

Did anybody else notify similar problems? I actually don't know how to debug this issue...

Thanks, Michael

**Comment by Farfar** [comment]
> Hi, since a few days the integrations stopped working and on startup it takes ages and slows down the full Home Assistant:
> 
> 2021-02-16 21:38:45 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: skodaconnect
> 2021-02-16 21:38:46 WARNING (MainThread) [homeassistant.setup] Setup of skodaconnect is taking over 10 seconds.
> 2021-02-16 21:39:45 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: skodaconnect
> 
> Logger: homeassistant.setup
> Source: setup.py:215
> First occurred: 21:43:36 (1 occurrences)
> Last logged: 21:43:36
> 
> Setup of skodaconnect is taking longer than 300 seconds. Startup will proceed without waiting any longer
> 
> Did anybody else notify similar problems? I actually don't know how to debug this issue...
> 
> Thanks, Michael

Have you enabled debug logging for the component and library? Start there and see where it stops.

**Comment by nanobyte84** [comment]
Hi,
I added:

logger:
    default: info
    logs:
        skodaconnect.connection: debug
        skodaconnect.vehicle: debug
        custom_components.skodaconnect: debug
        custom_components.skodaconnect.climate: debug
        custom_components.skodaconnect.lock: debug
        custom_components.skodaconnect.device_tracker: debug
        custom_components.skodaconnect.switch: debug
        custom_components.skodaconnect.binary_sensor: debug
        custom_components.skodaconnect.sensor: debug

and 
     response_debug: true


but I don't have more detailed debug information...


2021-02-16 22:57:56 DEBUG (MainThread) [skodaconnect.connection] Got authorization endpoint, logging on.
2021-02-16 22:57:56 DEBUG (MainThread) [skodaconnect.connection] Authenticating with email and password.
2021-02-16 22:57:59 WARNING (MainThread) [homeassistant.setup] Setup of skodaconnect is taking over 10 seconds.
...
2021-02-16 23:02:48 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: skodaconnect
2021-02-16 23:02:49 ERROR (MainThread) [homeassistant.setup] Setup of skodaconnect is taking longer than 300 seconds. Startup will proceed without waiting any longer

is there anything i can do to get more info?

**Comment by Farfar** [comment]
> Hi,
> I added:
> 
> logger:
> default: info
> logs:
> skodaconnect.connection: debug
> skodaconnect.vehicle: debug
> custom_components.skodaconnect: debug
> custom_components.skodaconnect.climate: debug
> custom_components.skodaconnect.lock: debug
> custom_components.skodaconnect.device_tracker: debug
> custom_components.skodaconnect.switch: debug
> custom_components.skodaconnect.binary_sensor: debug
> custom_components.skodaconnect.sensor: debug
> 
> and
> response_debug: true
> 
> but I don't have more detailed debug information...
> 
> 2021-02-16 22:57:56 DEBUG (MainThread) [skodaconnect.connection] Got authorization endpoint, logging on.
> 2021-02-16 22:57:56 DEBUG (MainThread) [skodaconnect.connection] Authenticating with email and password.
> 2021-02-16 22:57:59 WARNING (MainThread) [homeassistant.setup] Setup of skodaconnect is taking over 10 seconds.
> ...
> 2021-02-16 23:02:48 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: skodaconnect
> 2021-02-16 23:02:49 ERROR (MainThread) [homeassistant.setup] Setup of skodaconnect is taking longer than 300 seconds. Startup will proceed without waiting any longer
> 
> is there anything i can do to get more info?

Seems like something halts after final stage of authentication. Can you check the app, logout and in again to verify that your credentials are working.

Are you running HA core? In venv?
My sugestion is you uninstall the component and make sure all remnants of the library are removed as well, then reinstall.

**Comment by nanobyte84** [comment]
Yes I tried logging in and out successfully in a browser and the android app.
I also did not change the credentials etc.

I am running HA core in a virtual machine in PROXMOX...

**Comment by Farfar** [comment]
Unfortunately there's not enough debug info to determine whats going wrong.
So two (three) options. Either try uninstall/reinstall or I need to code some more debug logs.
There's also a third option if you feel confident enough to add some lines of code yourself?

**Comment by nanobyte84** [comment]
For me it would be no problem to add some code etc.
But I am on a new road now - I installed https://github.com/lendy007/skodaconnect.git on ubuntu and created a second account at myskoda.
With the CLI version of skodaconnect I can also not login to my account - so I changed the password and it still did not work.
My blank account on the other hand does login...
So I try now to migrate my car to the new account and hope it will work....
